src/statemachine/RSocketStateMachine.cpp

```diff
@@ -354,18 +364,23 @@ void RSocketStateMachine::closeStreams(StreamCompletionSignal signal) {
 }
 
 void RSocketStateMachine::pauseStreams() {
+  VLOG(4) << "Pausing streams.";
 //  for (auto& streamKV : streamState_->streams_) {
 //    streamKV.second->pauseStream(*requestHandler_);
 //  }
 }
 
 void RSocketStateMachine::resumeStreams() {
+  VLOG(4) << "Resuming streams.";
 //  for (auto& streamKV : streamState_->streams_) {
 //    streamKV.second->resumeStream(*requestHandler_);
 //  }
 }
 
 void RSocketStateMachine::processFrame(std::unique_ptr<folly::IOBuf> frame) {
+  VLOG_EVERY_N(2, 10000) << "processFrame - frame length=" << frame->length() << std::endl
```

Comment:
I think same for all of these, if I want this logging I want it for all frames, not just one every 10 seconds. Also 2 seems too low of a level for frame logging?